core/src/main/java/io/questdb/cutlass/pgwire/modern/PGPipelineEntry.java

```diff
@@ -1390,7 +1390,7 @@ private void msgExecuteSelect(
             }
 
             sqlExecutionContext.getCircuitBreaker().resetTimer();
-            sqlExecutionContext.setCacheHit(cacheHit);
+            sqlExecutionContext.setCacheHit(true);
```

Comment:
`msgExecuteSelect` is used in many modes, except for simple. We can't unconditionally set cache hit flag to true here as it could be the case that the factory wasn't in the query cache, but was just created.

Reply:
With this approach the very first query execution will print `cache=true` which is incorrect. Maybe it'd be simpler to add a long counter field for the number of `PGPipelineEntry` executions with the same factory instance?